sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/AbstractResourceMapper.java

```diff
@@ -0,0 +1,90 @@
+package com.mesosphere.sdk.offer.evaluate;
```

Comment:
Nice cleanup!

I think these could just be presented as static functions, in e.g. a `ResourceMapperUtils` class. The `resourceNamespace` member appears to only be used in one of the functions, anyway, and could just be passed in as a parameter. The SDK code has generally avoided inheritance.

Reply:
I think it'd be fine to have a static logger in the utils class

Reply:
Ping for static utils instead of inheritance